experiment/simulation/mild.html

```diff
@@ -0,0 +1,87 @@
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/virtual-labs/virtual-style@0.0.6-b/css/style.min.css" />
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+
+<div class="v-tabs ml-4">
+    <ul>
+        <li>
+            <a href="index.html">
+                <span>ASTM A53</span>
+            </a>
+        </li>
+        <li>
+            <a href="corten.html">
+                <span>Corten Steel</span>
+            </a>
+        </li>
+        <li class="is-active">
+            <a href="mild.html">
+                <span>Mild Steel</span>
+            </a>
+        </li>
+        <li>
+            <a href="comparison.html">
+                <span>Comparison</span>
+            </a>
+        </li>
+    </ul>
+</div>
+<div class="box">
+    <div class="columns is-centered">
+        <div class="column has-text-centered is-two-fifths mt-0">
+            <canvas class="main" id="main"></canvas>
+            <br>
+            <button id="play" class="v-button">Play</button>
+            <button id="pause" class="v-button" disabled>Pause</button>
+            <button id="restart" class="v-button" disabled>Restart</button>
+            <br>
+            <div class="slidecontainer">
+                <p>Simulation speed: <span id="demo_speed"></span>x</p>
+                <input type="range" min="1" max="8" value="4" class="slider" id="speed">
+            </div>
+        </div>
+        <div class="column has-text-centered">
+            <div class="v-table-wrap">
+                <table class="table is-bordered is-fullwidth v-table-primary mb-2">
+                    <thead>
+                        <tr>
+                            <th>Length <br> (mm)</th>
+                            <th>Diameter <br> (mm)</th>
+                            <th>Stress <br> (Mpa)</th>
+                            <th>Strain</th>
+                            <th>Load <br> (N)</th>
+                            <th>Elongation <br> (mm)</th>
+                            <th>Youngs Modulus <br> (Mpa)</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>350</td>
+                            <td>11.6</td>
+                            <td><span id="stress">0.0 </span></td>
+                            <td><span id="strain">0.0 </span></td>
+                            <td><span id="load">0.0</span></td>
+                            <td><span id="elongation">0.0</span> </td>
+                            <td><span id="young">0.0</span></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div>
+                <div id="chartContainer"></div>
+            </div>
+        </div>
+    </div>
+</div>
+<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>
+<script src="./js/mild.js"></script>
+<script src="./js/data.json"></script>
```

Comment:
Please follow the pattern of importing data.json before mild.js since it is used in mild.js